Resources/Private/Partials/Document/List.html

```diff
@@ -87,6 +87,14 @@
                     <span class="glyphicon glyphicon-eye-open"></span>
                 </dpf:link.preview>
 
+                <dpf:link.datacite
```

Comment:
The ViewHelper must be called <dpf:link.data**C**ite> (capital "C") as the viewHelper-Class ist called Data**C**iteViewHelper.

This is the reason for the TYPO3-Exception:

#1289386765: Could not analyse class:EWW\Dpf\ViewHelpers\Link\Data**c**iteViewHelper maybe not loaded or no autoloader? (More information)

Reply:
@ruegamer Can you please change this, and also `git rebase origin/master`?